inc/cachify.class.php

```diff
@@ -651,7 +651,7 @@ public static function add_dashboard_count( $items = array() )
 			esc_attr(strtolower($method)),
 			esc_html__( 'Caching method', 'cachify' ),
 			esc_attr($method),
-			( empty($size) ? esc_html__( 'Empty', 'cachify' ) : size_format($size) )
+			( empty($size) ? esc_html__( 'Empty Cache', 'cachify' ) : size_format($size) . ' ' . esc_html__( 'Cache', 'cachify' ) )
```

Comment:
think its more friendly i did because if we let _empty_ alone it can be translated to "leer"- Cache or "leerer" - Cache the right word by word translation without context is "leer" but it should be the second one. if we let cache alone it is a noun and there are less misinterpretation. Dont know how it will be in other languages.